package.json

```diff
@@ -20,15 +20,21 @@
     "url": "git://github.com/edwellbrook/node-tvdb.git"
   },
   "dependencies": {
-    "jszip": "3.1.2",
-    "request": "2.74.0",
-    "xml2js": "0.4.17"
+    "debug": "^2.2.0",
+    "lodash": "^4.15.0",
+    "request": "^2.74.0",
+    "request-promise": "^4.1.1"
```

Comment:
`request-promise-core` is not recommended as a dependency other that for a `request-promise-x` implementation.

Please see http://programmers.stackexchange.com/questions/278778/why-are-native-es6-promises-slower-and-more-memory-intensive-than-bluebird .

I suggest not to sacrifice better performance and memory usage to minimizing dependencies.